ts/store/actions/constants.ts

```diff
@@ -85,13 +85,43 @@ export const TRANSACTIONS_FETCHED: "TRANSACTIONS_FETCHED" =
   "TRANSACTIONS_FETCHED";
 export const SELECT_TRANSACTION_FOR_DETAILS: "SELECT_TRANSACTION_FOR_DETAILS" =
   "SELECT_TRANSACTION_FOR_DETAILS";
-// (cards)
-export const FETCH_CARDS_REQUEST: "FETCH_CARDS_REQUEST" = "FETCH_CARDS_REQUEST";
-export const CARDS_FETCHED: "CARDS_FETCHED" = "CARDS_FETCHED";
-export const SELECT_CARD_FOR_DETAILS: "SELECT_CARD_FOR_DETAILS" =
-  "SELECT_CARD_FOR_DETAILS";
-export const SET_FAVORITE_CARD: "SET_FAVORITE_CARD" = "SET_FAVORITE_CARD";
-export const START_PAYMENT: "START_PAYMENT" = "START_PAYMENT";
+// (wallets)
+export const FETCH_WALLETS_REQUEST: "FETCH_WALLETS_REQUEST" =
+  "FETCH_WALLETS_REQUEST";
+export const WALLETS_FETCHED: "WALLETS_FETCHED" = "WALLETS_FETCHED";
+export const SELECT_WALLET_FOR_DETAILS: "SELECT_WALLET_FOR_DETAILS" =
+  "SELECT_WALLET_FOR_DETAILS";
+export const SET_FAVORITE_WALLET: "SET_FAVORITE_CARD" = "SET_FAVORITE_CARD";
```

Comment:
`SET_FAVORITE_WALLET`

Reply:
Isn't it what's already there? If you're referring to the different spellings (UK vs US), we should probably define which one we should use and stick with it throughout the code (and this concerns all words that can be spelled multiple ways)

Reply:
`export const SET_FAVORITE_WALLET: "SET_FAVORITE_CARD" = "SET_FAVORITE_CARD";` -> `export const SET_FAVORITE_WALLET: "SET_FAVORITE_WALLET" = "SET_FAVORITE_WALLET";`

Reply:
Totally missed the `CARD` part, thanks!